armi/bookkeeping/db/tests/test_databaseInterface.py

```diff
@@ -343,11 +348,11 @@ def test_growToFullCoreFromFactoryWithCS(self):
 
         r.core.growToFullCore(self.cs)
 
-        self.assertEqual(r.core.numRings, 9)
+        self.assertEqual(r.core.numRings, 3)
         self.assertEqual(r.p.cycle, 0)
-        self.assertEqual(len(r.core.assembliesByName), 217)
+        self.assertEqual(len(r.core.assembliesByName), 19)
         self.assertEqual(len(r.core.circularRingList), 0)
-        self.assertEqual(len(r.core.blocksByName), 1085)
+        self.assertEqual(len(r.core.blocksByName), 95)
```

Comment:
It could be pretty useful if we made all of these expected values into class variables in the `setUpClass()`, just to avoid the repetition.

We could even just define a little helper method with this block of code in it, to avoid repeating all of that entirely.

Reply:
Not necessary, of course, just a suggestion.

Reply:
Sure. Done.